data/sql/updates/pending_db_world/rev_1606636935711383900.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606636935711383900');
+
+-- Apply 'Shadowform' Aura to Saronite Animus
+
+UPDATE `creature_template_addon` SET `auras` = '63359' WHERE (`entry` = 33524);
+UPDATE `creature_template_addon` SET `auras` = '63359' WHERE (`entry` = 34152);
```

Comment:
ints

```suggestion
UPDATE `creature_template_addon` SET `auras` = 63359 WHERE (`entry` = 33524);
UPDATE `creature_template_addon` SET `auras` = 63359 WHERE (`entry` = 34152);
```

Reply:
actually you are wrong silker

This table is a string so it should have quotes.